src/main/java/uk/gov/hmcts/reform/iahomeofficemockapi/generated/infrastructure/api/ApplicationStatusApi.java

```diff
@@ -215,4 +138,38 @@
 
     }
 
+    @NotNull
+    private String getResponseJson(String homeOfficeReference) throws IOException {
+
+        String responseJsonFile;
+
+        switch (homeOfficeReference) {
+            case "1212-0099-0061-7075": // Valid UAN reference
+                responseJsonFile = "ho-api_get-by-search-params_200-valid_response-01.json";
+                break;
+            case "1212-0099-0062-8083":
+                responseJsonFile = "ho-api_get-by-serach-params_200_valid-response-02.json";
+                break;
+            case "020639760": // CID reference response
+                responseJsonFile = "ho-api_get-by-search-params_200-valid_response-01.json";
+                break;
+            case "020651434": // CID reference - not Asylum case response
+                responseJsonFile = "ho-api_get-by-search-params_200-valid_response-01.json";
+                break;
+            case "1212-0099-0029-3029": // Valid UAN - no data found
+                responseJsonFile = "ho_api_get-by-search-params_200_no-data-found-uan-ref.json";
+                break;
+            case "123123123": // Valid CID - no data found
+                responseJsonFile = "ho_api_get-by-search-params_200_no-data-found-uan-ref.json";
+                break;
+            default:
```

Comment:
For default, wouldn't it be better to send some valid 400 response instead of exception?

Reply:
Have added to keep it different to the 400 response we expect. It is more of a invalid test configuration from our side than a bad request to the validation api. That was my thinking.

Reply:
ok